blueprints/app/files/testem.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   test_page: 'tests/index.html?hidepassed',
   disable_watching: true,
+  browser_start_timeout: 60000,
```

Comment:
I don't understand the particulars yet, but there's a race condition between testem starting up and the browser connecting. The default timeout is occasionally not a sufficient wait, and the end result is having to kick the job again to get a successful test run

Reply:
Ah nice, I was struggling with this, but it seems to only error on ubuntu and doesn't on OS X